tools/test/config/feature_recursive_add/test_data.json

```diff
@@ -1,6 +1,6 @@
 {
     "test_target": {
-        "expected_features": ["BOOTLOADER", "STORAGE"],
-        "included_source": ["FEATURE_BOOTLOADER/lib1/lib1.c", "FEATURE_STORAGE/lib2/lib2.c"]
+        "expected_features": ["BOOTLOADER"],
```

Comment:
Please don't remove code that tests how the tools work WRT features just because the feature that's used as test data is no longer part of Mbed OS

Reply:
I don't mind reverting this file, but will this work even if STORAGE feature and FEATURE_STORAGE directory no longer exist?
Alternatively, I can replace these with other feature and directory, if this only requires any two features.

Reply:
Yes. it uses a local folder. You're welcome to rework it to not use the string "STORAGE" if you're so inclined, but I don't think it's worth the time.

Reply:
Thanks. Indeed got better things to do with my time. Reverted.